b0tweb/IRCConnectionController.cs

```diff
@@ -35,7 +36,7 @@ class IRCConnectionController
         /// <summary>
         /// The proxy host port of the Tor proxy server.
         /// </summary>
-        private const int ProxyPort = 9050;
+        private const int ProxyPort = 9150;
```

Comment:
Tor is supposed to start up from the binary. Then the port is 9050, at least on my machine it was the case. It does not work for you that way?

Reply:
I have changed the diff back to what it was before. Since that's how it should be. But no, for some reason it wasn't connecting to the server. It wasn't throwing any errors either and the irc client was 'listening' but no bot was on the channel. will look in to this more another day.